aiosmtpd/tests/test_handlers.py

```diff
@@ -38,8 +38,7 @@ def setUp(self):
     def test_debugging(self):
         with ExitStack() as resources:
             client = resources.enter_context(SMTP(*self.address))
-            resources.enter_context(
-                patch('aiosmtpd.handlers._format_peer', _format_peer_address))
+            peer = client.sock.getsockname()
```

Comment:
I don't mind this change, but there are two places in test_handlers.py where we mock `_format_peer`.  I think I'll just fix this up after landing your branch.